### PR TITLE
MGT-1: Updated the script with management vrf name and added deviation for table-connection config

### PR DESCRIPTION
**Pull request by mohanasm**
1. Updated the script with management vrf name for Juniper (Refer b/271899732#comment46)
2. Updated traffic verification with flow metrics and interface statistics to avoid counting control packets.

### CODE REVIEW

File: feature/system/management/otg_tests/management_ha_test/management_ha_test.go [hunk 1]
```diff
@@ -54,8 +54,24 @@ var (
...
+		ondatra.NOKIA:   "mgmtvrf1",
+	}
+	loopbackIntf = map[ondatra.Vendor]int{
+		ondatra.JUNIPER: 0,
+		ondatra.ARISTA:  1,
+		ondatra.CISCO:   1,
+		ondatra.NOKIA:   1,
+	}
+	loopbackSubIntf = map[ondatra.Vendor]int32{
+		ondatra.JUNIPER: 0,
```
Comment (Swetha-haridasula) [inline feature/system/management/otg_tests/management_ha_test/management_ha_test.go:70]: As per design I believe Juniper is using subinterface of 100. Please check

Comment (mohanasm) [reply feature/system/management/otg_tests/management_ha_test/management_ha_test.go]: Updated it in latest commit. Please check.

File: feature/system/management/otg_tests/management_ha_test/management_ha_test.go [hunk 2]
```diff
@@ -54,8 +54,24 @@ var (
 		IPv4Len: 32,
 		IPv6Len: 128,
 	}
-
-	mgmtVRF  = "mvrf1"
+	mgmtVRF = map[ondatra.Vendor]string{
+		ondatra.JUNIPER: "mgmt_junos",
```
Comment (Swetha-haridasula) [inline feature/system/management/otg_tests/management_ha_test/management_ha_test.go:58]: Hi Mohana, as we are using this vrf for device reachability, we are looding access to device in middle of test. can we use a different vrf name here please

File: feature/system/management/otg_tests/management_ha_test/management_ha_test.go [hunk 3]
```diff
@@ -54,8 +54,24 @@ var (
...
+		ondatra.NOKIA:   "mgmtvrf1",
+	}
+	loopbackIntf = map[ondatra.Vendor]int{
+		ondatra.JUNIPER: 0,
+		ondatra.ARISTA:  1,
+		ondatra.CISCO:   1,
+		ondatra.NOKIA:   1,
+	}
+	loopbackSubIntf = map[ondatra.Vendor]int32{
+		ondatra.JUNIPER: 100,
```
Comment (Swetha-haridasula) [inline feature/system/management/otg_tests/management_ha_test/management_ha_test.go:70]: change of subinterface to be 10 as well

File: feature/system/management/otg_tests/management_ha_test/management_ha_test.go (hunk 2 above)
Comment (mohanasm) [reply feature/system/management/otg_tests/management_ha_test/management_ha_test.go]: Hi Swetha, Updated the vrf name.Please check.

File: feature/system/management/otg_tests/management_ha_test/management_ha_test.go (hunk 3 above)
Comment (mohanasm) [reply feature/system/management/otg_tests/management_ha_test/management_ha_test.go]: Done. Please check.

**Review by Swetha-haridasula — APPROVED** [review]
lgtm

**Pull request merged**